examples/ensure-name-substring/ensure-name-substring-prefix/README.md

```diff
@@ -5,6 +5,14 @@
 This example demonstrates how to declaratively run the [`ensure-name-substring`]
 function to prepend prefix in the resource names.
 
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/ensure-name-substring/ensure-name-substring-prefix
```

Comment:
you don't need the intermediate directory any more:

`examples/<function-name>-<example-name>`

Reply:
We can remove the intermediate directory, but I need to modify the site deployment script.

Reply:
@etefera is going to help with it.
It is tracked in https://github.com/GoogleContainerTools/kpt/issues/2250